apis/userapi/userapi.yaml

```diff
@@ -1937,78 +1950,234 @@ paths:
+                        userType: student
                         status: 1
-                        lastName: selfdeclaredev4
-                        roles:
-                          - PUBLIC
-                        prevUsedPhone: ''
+                        lastName: null
+                        tncLatestVersion: 3.5.0
+                        roles:  PUBLIC
+                        prevUsedPhone:
                         stateValidated: false
                         isDeleted: false
                         organisations:
-                          - updatedBy:
-                            organisationId: 0126796199493140480
-                            orgName: Pre-prod Custodian Organization
-                            addedByName:
-                            addedBy:
-                            roles:
-                              - PUBLIC
-                            approvedBy:
-                            updatedDate:
-                            userId: ec8deeb2-4ded-4fa2-ac48-023ad8298d92
-                            approvaldate:
+                          - organisationId: 0126796199493140480
+                            updatedBy: null
+                            orgName: Staging Custodian Organization
+                            addedByName: null
+                            addedBy: null
+                            roles: PUBLIC
+                            approvedBy: null
+                            channel: dikshapreprodcustodian
+                            locationIds:
+                              027f81d8-0a2c-4fc6-96ac-59fe4cea3abf
+                              8250d58d-f1a2-4397-bfd3-b2e688ba7141
+                            orgLocation:
+                              - type: state
+                                id: 027f81d8-0a2c-4fc6-96ac-59fe4cea3abf
+                              - type: district
+                                id: 8250d58d-f1a2-4397-bfd3-b2e688ba7141
+                            externalId: 101010
+                            updatedDate: null
+                            userId: 96102c3f-2c22-4614-8dcc-6b130cefe586
+                            approvaldate: null
+                            isSchool: false
                             isDeleted: false
                             hashTagId: 0126796199493140480
-                            isRejected:
-                            position:
-                            id: 0131573015167221767
-                            orgjoindate: 2020-11-23 07:02:43:194+0000
-                            isApproved:
-                            orgLeftDate:
-                        provider:
-                        countryCode:
-                        maskedEmail: se*************@yopmail.com
-                        email: se*************@yopmail.com
+                            isRejected: null
+                            locations:
+                              - code: 29
+                                name: Karnataka
```

Comment:
@sreeragksgh @mohitga request that we do not use actual names, etc. in Sunbird documentation. Can we replace 'Karnataka' 'Belgavi' etc. with variable names providing these as examples? This is a global comment. Further down there are references to other districts, States and towns too.